### PR TITLE
Update README for XCFramework support

### PR DESCRIPTION
**Pull request by zhinang-amazon**
### Description of changes:
Update .gitignore and README for XCFramework support
refer binary download to release tab instead of directly on README

#### Manual test cases (add more as needed):

- [ ] Join meeting without CallKit
- [ ] Join meeting with CallKit as Incoming
- [ ] Join meeting with CallKit as Outgoing
- [ ] Leave meeting
- [ ] Rejoin meeting
- [ ] See metrics
- [ ] See attendee presence
- [ ] Send audio
- [ ] Receive audio
- [ ] Mute self
- [ ] Unmute self
- [ ] See local mute indicator when muted
- [ ] See remote mute indicator when other is muted
- [ ] Enable and disable local video
- [ ] Enable and disable remote video from remote side
- [ ] Send local video
- [ ] Pause and resume remote video
- [ ] Switch local camera
- [ ] Receive remote screen share

#### Integration validation (required before release)

- [ ] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

### Screenshots, if available:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,27 +35,23 @@ And review the following guides:
 
 To include the SDK binaries in your own project, follow these steps.
 
-For the purpose of setup, your project's root folder (where you can find your `.xcodeproj` file) will be referred to as `root`
+For the purpose of setup, your project's root folder (where you can find your `.xcodeproj` file) will be referred to as `root`.
 
 ### 1. Download binaries
 
-Download the following zips:
```
Comment (linsang21) [inline README.md:42]: ❤
update to the runbook?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+### Added
+* Added support for XCFramework. AmazonChimeSDK and AmazonChimeSDKMedia binaries now contain .xcframework format as well. See `README.md` for updated Setup instructions.
```
Comment (linsang21) [inline CHANGELOG.md:4]: Is this only document change? Should we add [Document] prefix?

File: README.md (hunk 1 above)
Comment (zhinang-amazon) [reply README.md:42]: Will do afterwards. I will update Android too

File: CHANGELOG.md (hunk 2 above)
Comment (zhinang-amazon) [reply CHANGELOG.md:4]: This is not just documentation though, there are some build changes and XCFrameworks are in the binaries now that builders can choose between .framework and .xcframework.

**Review by linsang21 — APPROVED** [review]

**Review by hokyungh — APPROVED** [review]

**Pull request merged**